net/unbound/files/unbound.sh

```diff
@@ -615,7 +615,7 @@ unbound_forward() {
       for fdomain in $UNBOUND_LIST_FORWARD ; do
         {
           echo "forward-zone:"
-          echo "  name: \"$fdomain.\""
+          echo "  name: \"$fdomain\""
```

Comment:
There are a few other places Unbound doc has terminating period, but it is not required. I think they are in other parts of the script, and should be pruned. _(I guess that is what happens when you read the directions.)_